Aztec/Classes/TextKit/TextStorage.swift

```diff
@@ -231,6 +233,13 @@ open class TextStorage: NSTextStorage {
 
         return textStore.attributes(at: location, effectiveRange: range)
     }
+
+    private func replaceTextStoreString(_ range: NSRange, with string: String) {
```

Comment:
Considering `String.replaceSubrange()` is of complexity `O(m)` for most scenarios (where m is the sum of the length of the cached string and the modification), this method should be replaced by:

```
textStoreString = textStore.string
```